disclosure_backend/management/commands/loadalldata.py

```diff
@@ -0,0 +1,21 @@
+import glob
+import os
+from django.conf import settings
+from django.core.management import call_command
+from calaccess_raw.management.commands import CalAccessCommand
+
+
+class Command(CalAccessCommand):
+    help = 'Generate documentation for raw CAL-ACCESS database models'
```

Comment:
Copy-paste error :) And actually, this is supposed to be an extension of Django's `loaddata` command, which loads a specified fixture. So hmmm, maybe I need a different name.